cigar/fancy.go

```diff
@@ -14,6 +14,11 @@ func MakeExplicit(cigar []*Cigar, sequence []dna.Base, reference []dna.Base) []*
 	var answer []*Cigar
 	var seqIdx, refIdx int
 	var k, runLenCount int64
+
+	if cigar[0].Op == '*' {
+		return []*Cigar{&Cigar{Op: '*'}}
```

Comment:
I think Eric weighed in here saying that the original version (return nil) is correct and I think my earlier comment threw us off.  Seems like there are two ways to do it currently in the code, but we are going to try to converge on the "nil = unaligned"

Reply:
Switched it back to nil